src/main/kotlin/com/blincke/commune_api/repositories/FriendshipRepository.kt

```diff
@@ -2,6 +2,16 @@ package com.blincke.commune_api.repositories
 
 import com.blincke.commune_api.models.database.friends.Friendship
 import com.blincke.commune_api.models.database.friends.FriendshipId
+import com.blincke.commune_api.models.database.users.User
 import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Query
 
-interface FriendshipRepository : JpaRepository<Friendship, FriendshipId>
+interface FriendshipRepository : JpaRepository<Friendship, FriendshipId> {
+    @Query(
+        value = "select f from Friendship f where " +
+                "(f.friendshipId.requester = (?1) and f.friendshipId.recipient = (?2))" +
+                "or " +
+                "(f.friendshipId.recipient = (?1) and f.friendshipId.requester = (?2))"
+    )
+    fun selectRelevantFriendships(user1: User, user2: User): List<Friendship>
```

Comment:
This should just return a list of length 0 1 or 2 - probably a better way of doing it in sql, but I don't think I know how. This works though.